.github/workflows/android_cmake/start.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+set -e
+
+apt-get update -y
+
+# pkg-config sqlite3 for proj compilation
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    wget unzip ccache curl ca-certificates \
+    pkg-config make binutils sqlite3 \
+    automake
+
+cd "$WORK_DIR"
+
+if test -f "$WORK_DIR/ccache.tar.gz"; then
+    echo "Restoring ccache..."
+    (cd $HOME && tar xzf "$WORK_DIR/ccache.tar.gz")
+fi
+
+# We need a recent cmake for recent NDK versions
+wget -q https://github.com/Kitware/CMake/releases/download/v3.22.3/cmake-3.22.3-linux-x86_64.tar.gz
+tar xzf cmake-3.22.3-linux-x86_64.tar.gz
+export PATH=$PWD/cmake-3.22.3-linux-x86_64/bin:$PATH
+
+# Download Android NDK
+wget -q https://dl.google.com/android/repository/android-ndk-r23b-linux.zip
+unzip -q android-ndk-r23b-linux.zip
+
+export ANDROID_NDK=$PWD/android-ndk-r23b
+export NDK_TOOLCHAIN=$ANDROID_NDK/toolchains/llvm/prebuilt/linux-x86_64
+
+ccache -M 1G
+ccache -s
+
+# build sqlite3
+wget -q https://sqlite.org/2022/sqlite-autoconf-3370200.tar.gz
+tar xzf sqlite-autoconf-3370200.tar.gz
+cd sqlite-autoconf-3370200
+CC="ccache $NDK_TOOLCHAIN/bin/aarch64-linux-android24-clang" ./configure \
+  --prefix=/tmp/install --host=x86_64-linux-android24
+make -j3
+make install
+cd ..
```

Comment:
Hi @rouault, I've successfully built gdal, proj and sqlite3 same way as You did here. However when I run android application with these libraries I get this error:
```
AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: library "libsqlite3.so.0" not found: needed by /data/app/~~X1wOeY5-90cImKIsrkJNMQ==/com.myapp.mycompany-_udxwh7uKcyKcoksBRfj7g==/lib/arm64/libgdal.so in namespace classloader-namespace
```
while `libsqlite3.so` is deployed to the APK bundle at the same location as `libgdal.so`: `/lib/arm64-v8a`. Any ideas what is the problem and how to solve?

Reply:
I know this, and this is the 2nd question: why it asks for `libsqlite3.so.0` and not for `libsqlite3.so`. May be some wrong or missed `cmake` build arguments? How to solve the situation?

Reply:
I would start with checking the logs and installed files from the sqlite3 build. Does it properly use the Android naming conventions?

But first, everything is targeting arm64, but the sqlite build is configured for `--host=x86_64-linux-android24` 💥

Reply:
no, in my case it is configured for `aarch64-linux-android24`.

Reply:
Adding `LDFLAGS="-avoid-version"` to `./configure` [solves](https://stackoverflow.com/a/12067374/630169) the issue. No error now, however application crashes, not sure it is connected with GDAL build however.